packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1790,6 +1804,23 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
     }
 
 
+    /********************************
+     * Internal Functions: Upgrades *
+     ********************************/
+    
+    function _isUpgrading()
+        internal
+        returns (
+            bool
+        )
+    {
+        return uint256(_getContractStorage(
```

Comment:
An alternative to this approach here is `ovmCALL(0x420000000000000000000000000000000000000D, abi.encodeWithSelector(L2ChugSplashDeployer.isUpgrading, gasleft())`.  FWIW, this would allow for de-duplicating the storage slot, although in this case the benefit feels minimal.